src/rules/NoIgnoredReturnRule.ts

```diff
@@ -40,7 +40,8 @@ class Walker extends Lint.ProgramAwareRuleWalker {
             "filter",
             "findIndex",
             "keys",
-            "map",
+            // exclude "map" as it's often used in place of "forEach"
```

Comment:
For me this is not good enough reason to exclude `map`. There are people that writing good code and use `map` and `forEach` for their purposes, and they want to benefit from the full power of the rule.

 At least we could provide an option for that.